src/Api/Issue.php

```diff
@@ -153,4 +153,9 @@ public function timeTracking(): TimeTracking
     {
         return new TimeTracking($this->getClient());
     }
+
+    public function command($parameters)
+    {
+        return $this->post("commands", $parameters);
+    }
```

Comment:
Is a command really always part of an issue? I think we can remove this part again and go straight to the "Commands" entry point.